src/executables/uber-time.js

```diff
@@ -9,12 +9,22 @@ import CommandExecutionService from '../services/CommandExecutionService';
 const service = new CommandExecutionService();
 
 program
-  .arguments('<address>')
-  .action((address) => {
-    try {
-      service.executeTimeEstimates(address).then(table => console.log(table));
-    } catch (Error) {
-      console.error('Could not get time estimates');
-    }
-  })
+  .usage('<address>')
```

Comment:
Because Arguments doesn't really do what you need it to in this situation. Or at least, it doesn't really do *anything*. See this example: https://github.com/tj/commander.js/blob/master/examples/pm-install#L9
Even the writer of the library just goes straight for using the args passed to the command.